.gitattributes

```diff
@@ -34,4 +34,3 @@
 # Avoid merge conflicts in CHANGELOG
 # https://about.gitlab.com/2015/02/10/gitlab-reduced-merge-conflicts-by-90-percent-with-changelog-placeholders/
 /CHANGELOG.md		merge=union
-
```

Comment:
It is a good practice to leave empty lines in the end of file.
Why do you think vice versa?

Reply:
Now two empty lines in the end of this file. I'm leave only one empty line.

Reply:
As I see you removed it.
![image](https://user-images.githubusercontent.com/6815714/122000432-79bbeb80-cdb7-11eb-9f32-2a9ef47eeae0.png)

Reply:
GitHub don't show last empty line. See any PHP file. For example: https://github.com/yiisoft/html/blob/master/src/NoEncodeStringableInterface.php

When file don't have last empty line github show icon:
![image](https://user-images.githubusercontent.com/525501/122002275-344ced80-cdba-11eb-8b21-a999563d6eab.png)